bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
I found the culprits:
* https://github.com/zeromq/zproject/blob/master/zproject_class_api.gsl#L386
* https://github.com/zeromq/zproject/blob/master/zproject_class_api.gsl#L397
@bluca you can probably explain the indent here.